### PR TITLE
[ready] The good old batong 

### PR DESCRIPTION
**Pull request by DaddyIssues98**
## About The Pull Request

The batong is a "meme" toy. made with love so we all can recharge the batong in a sec official.

The item is just a toy, with no real impact in combat, just a simple toy with a stunbatong sound. 

## Why It's Good For The Game

the batong is a toy that everyone in the universe loves, is funny, you can do pranks to sec, and the idea of it was made in a fulp moment (TM) so we all love it. 

The batong can be purchased  from the vending machine "games"  this will make the crew visit the library more often.

## Changelog
:cl:
 Added batong.
 Added batong sound.
 Added batong icon.
/:cl:

### CODE REVIEW

File: fulp_modules/batong/code/arcade_thing.dm [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/obj/machinery/computer/arcade/Initialize()
+	. = ..()
+	if(GLOB.arcade_prize_pool)
+	GLOB.arcade_prize_pool += list(/obj/item/toy/plush/batong = 1)
```
Comment (JohnFulpWillard) [inline fulp_modules/batong/code/arcade_thing.dm:4]: add an extra tab on this line

File: fulp_modules/batong/code/batongcode.dm [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (JohnFulpWillard) [inline fulp_modules/batong/code/batongcode.dm:1]: I dont see why these 2 empty lines of code need to be here

File: fulp_modules/batong/code/batongcode.dm [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+/obj/item/toy/plush/batong
+	name = "\improper batong"
```
Comment (JohnFulpWillard) [inline fulp_modules/batong/code/batongcode.dm:4]: Why is this \improper?

Comment (Homingpenguins) [reply fulp_modules/batong/code/batongcode.dm:4]: i believe that was my fault, i probably gave him the wrong explanation reguarding the use of improper in the name.

File: fulp_modules/batong/code/batongcode.dm [hunk 4]
```diff
@@ -0,0 +1,11 @@
+
+
+/obj/item/toy/plush/batong
+	name = "batong"
+	desc = "A cheaply made toy. looks like it need some recharge"
+	icon = 'fulp_modules/batong/icon/toybatong.dmi'
+	icon_state = "toy"
+	inhand_icon_state = "toy"
+	attack_verb_continuous = list("stung")
```
Comment (ATH1909) [inline fulp_modules/batong/code/batongcode.dm:9]: stung?

File: fulp_modules/batong/code/batongcode.dm [hunk 5]
```diff
@@ -0,0 +1,11 @@
+
+
+/obj/item/toy/plush/batong
+	name = "batong"
+	desc = "A cheaply made toy. looks like it need some recharge"
```
Comment (ATH1909) [inline fulp_modules/batong/code/batongcode.dm:5]: is the lack of capitalization here intentional?

File: fulp_modules/batong/code/arcade_thing.dm [hunk 6]
```diff
@@ -0,0 +1,5 @@
+/obj/machinery/computer/arcade/Initialize()
+	. = ..()
+	if(GLOB.arcade_prize_pool)
+		GLOB.arcade_prize_pool += list(/obj/item/toy/plush/batong = 1)
+	///code made by Jhon fucking Willard.
```
Comment (ATH1909) [inline fulp_modules/batong/code/arcade_thing.dm:5]:
```suggestion
	///code made by John fucking Willard.
```

File: fulp_modules/batong/code/arcade_thing.dm [hunk 7]
```diff
@@ -0,0 +1,5 @@
+/obj/machinery/computer/arcade/Initialize()
+	. = ..()
+	if(GLOB.arcade_prize_pool)
+		GLOB.arcade_prize_pool += list(/obj/item/toy/plush/batong = 1)
```
Comment (ATH1909) [inline fulp_modules/batong/code/arcade_thing.dm:4]: are you 100% sure that you want this code to be run every time an arcade machine is created? what happens if the arcade machines are created before the arcade prize pool is? are you sure that this won't load the toy prize pool with tons of duplicate batong entries and/or heavily weight a single batong entry, or does += not work like that for lists?

File: fulp_modules/batong/code/batongcode.dm (hunk 4 above)
Comment (DaddyIssues98) [reply fulp_modules/batong/code/batongcode.dm:9]: like stun but with a g, like baton batong. but problably i'm going to change that text and the description. i was very tired when i wrote those lines.

File: fulp_modules/batong/code/arcade_thing.dm (hunk 7 above)
Comment (DaddyIssues98) [reply fulp_modules/batong/code/arcade_thing.dm:4]: turns out that  GLOB.arcade_prize_pool += list overwrites the prize pool with what is between (), i can fix it with adding all the prizes to the list that overwrites, and then it works fine

Comment (JohnFulpWillard) [reply fulp_modules/batong/code/arcade_thing.dm]: It's on the vending machine's initialize, it shouldn't have that problem

Comment (DaddyIssues98) [reply fulp_modules/batong/code/arcade_thing.dm]: if you make a arcade it give more % to get the baton, in one point you will get only batongs, so i change the batong to vending machines.

**Comment by EgorDinamit** [comment]
Honestly, I expected a harmless baton that you can actually recharge and run around scaring people...

**Comment by DaddyIssues98** [comment]
> Honestly, I expected a harmless baton that you can actually recharge and run around scaring people...

i had the same idea but people would  break into Ssec checkpoints, sci and/or robotics just to do the joke that they recharged the batong, so i made it a toy, with a similar sound and sprite.

File: fulp_modules/batong/code/batongcode.dm [hunk 8]
```diff
@@ -0,0 +1,10 @@
+/obj/item/toy/plush/batong
+	name = "Batong"
+	desc = "A cheaply made toy. Looks like it need some recharge maybe security can help you"
+	icon = 'fulp_modules/batong/icon/toybatong.dmi'
+	icon_state = "toy"
+	inhand_icon_state = "toy"
+	attack_verb_continuous = list("Tries to recharge the batong in")
+	attack_verb_simple = list("try to recharge the baton in")
```
Comment (ATH1909) [inline fulp_modules/batong/code/batongcode.dm:8]: *batong

File: fulp_modules/batong/code/batongcode.dm [hunk 9]
```diff
@@ -0,0 +1,10 @@
+/obj/item/toy/plush/batong
+	name = "Batong"
+	desc = "A cheaply made toy. Looks like it need some recharge maybe security can help you"
+	icon = 'fulp_modules/batong/icon/toybatong.dmi'
+	icon_state = "toy"
+	inhand_icon_state = "toy"
+	attack_verb_continuous = list("Tries to recharge the batong in")
```
Comment (ATH1909) [inline fulp_modules/batong/code/batongcode.dm:7]: *tries

File: tgstation.dme [hunk 10]
```diff
@@ -1,3343 +1,3345 @@
-// DM Environment file for tgstation.dme.
```
Comment (JohnFulpWillard) [inline tgstation.dme:1]: what

**Pull request closed without merging**